packages/node/README.md

```diff
@@ -62,9 +62,9 @@ yarn add @novu/node
 ```ts
 import { Novu } from '@novu/node';
 
-const novu = new Novu(process.env.NOVU_SECRET_KEY);
+const novu = new Novu('<NOVU_SECRET_KEY>');
```

Comment:
I think most folks will make use the previous of `process.env.NOVU_SECRET_KEY` pattern, this is the same pattern we follow elsewhere in our documentation. Was there a specific reason for this change?

Reply:
That makes sense for this PR, thanks for clarifying. We should align on a pattern everywhere in future docs updates. I prefer `new Novu(process.env.NOVU_SECRET_KEY);` as it's workable out of the box.